### ISSUE TITLE
[wg/math] Math Group Charter

### ISSUE THREAD

**Issue by bert-github** [w3c/strategy#545]
New charter proposal, reviewers please take note.

# Charter Review

[DRAFT Math Working Group Charter](https://w3c.github.io/mathml-docs/charter-2026.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/mathml-docs/charter-2026.html)

Expected end of charter refinement phase: [2026-06-14](https://lists.w3.org/Archives/Member/w3c-ac-members/2026AprJun/0004.html)

If applicable:

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FMath%2FDocuments%2FCharter2023.html&doc2=https%3A%2F%2Fw3c.github.io%2Fmathml-docs%2Fcharter-2026.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/math)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [ ] New WG 
 - [ ] New IG
 
* Existing 
 - [X] Existing WG recharter 
 - [ ] Existing IG recharter
 
## Substantive changes

‘MathML Core level 2’ is demoted to ‘tentative deliverable’.

Ditto for the requirements for MathML 5.

‘XML Entity Definitions for Characters’, previously mentioned as in scope for a future charter is now explicitly a deliverable. And, related to this, the Unicode Consortium is now mentioned as an organization to coordinate with.

There is additional auxiliary material under ‘other deliverables’, including software and a suggested mapping of LaTeX to accessible MathML.

## Communities suggested for outreach

See the section [‘Coordination’](https://w3c.github.io/mathml-docs/charter-2026.html#coordination) in the draft charter.

## Known or potential areas of concern

The group requests that issues on the charter be raised in the w3c/mathml repository of the Math WG: https://github.com/w3c/mathml/issues

## Anything else we should think about as we review? 

Note: proposed chairs should be copied @... on this issue: @NSoiffer, @bkardell

Note: The Technical Strategy Team Lead or Project & Process Team Lead will assign the issue to a [Charter Facilitator](https://www.w3.org/guide/process/charter.html#cr-acceptance) for new charters. For rechartering, the team contact is the Charter Facilitator by default, please assign the issue to them directly.

**Comment by himorin** [comment]
No comment or request from i18n

**Comment by matatk** [comment]
APA [discussed this on our call last week, and have no concerns](https://www.w3.org/2026/04/15-apa-minutes.html#9cd7); thanks!

Side note: charter review requests are duplicated in HR groups' `*-request` repos - I appreciate that is intended to help, giving us a 'heads up' on charters that are coming up for review, but we already check in here for the `charter` and `Horizontal review requested` labels, which is easy enough to do. Having the issues duplicated just gives us two places we need to comment in order to respond. I'd be interested in whether the other HR groups find them helpful, but APA would prefer to opt out of the duplicates please.

**Comment by npdoty** [comment]
No comments or issues from privacy.

**Comment by tidoust** [comment]
While the charter extension announcement pointed at the draft charter under preparation, [charter refinement](https://www.w3.org/guide/process/charter.html#charter-refinement) has not been formally announced. @bert-github, could you work with Comm to initiate charter refinement as soon as possible?

**Comment by simoneonofri** [comment]
No issues from Security

**Comment by tidoust** [comment]
> While the charter extension announcement pointed at the draft charter under preparation, [charter refinement](https://www.w3.org/guide/process/charter.html#charter-refinement) has not been formally announced. [@bert-github](https://github.com/bert-github), could you work with Comm to initiate charter refinement as soon as possible?

The charter extension announcement was not an explicit "charter review notice" but it linked to the draft charter, invited people to provide comments, and implicitly set a duration and identified a person responsible for the development of the Charter. Also, no substantive issues with the draft charter have been raised so far. In the interest of time, let's consider that the extension announcement fulfilled the role of a charter review notice, and let's move on with next step.

**Comment by caribouW3** [comment]
I find it slightly confusing that Core no longer has "level 1" but a "level 2" is listed in tentative deliverable.
I suppose that what's tentative is integrating those features into Core. or would it really be a separate document?

**Comment by bkardell** [comment]
> I suppose that what's tentative is integrating those features into Core. or would it really be a separate document?

We currently have additional feature ideas and things that are not part of the current spec and won't be added before moving forward.  We are referring to them as levels, but I don't expect this to matter in practice except for W3C transition purposes. In practice there is only "MathML-Core".

**Comment by plehegar** [comment]
[Proposed W3C Charter: Math Working Group (until 2026-08-07)](https://lists.w3.org/Archives/Public/public-new-work/2026Jul/0000.html)

**Comment by tidoust** [comment]
[Approved](https://lists.w3.org/Archives/Public/public-new-work/2026Aug/0012.html).
